Makefile

```diff
@@ -1,4 +1,4 @@
-all: checkmakefiles
+all: checkmakefiles makefiles
```

Comment:
makefile generateion should not be added to the all target. The reason is that the ALL target is called also from the IDE adn the IDE has its own makefile generator so the IDE will generate a proper makefile and then the makefile will overwrite it immediately.